lib/sdk/lang/type.js

```diff
@@ -126,7 +126,7 @@ exports.isGenerator = isGenerator;
  *    isArray({ 0: 'foo', length: 1 }) // false
  */
 var isArray = Array.isArray || function isArray(value) {
-  Object.prototype.toString.call(value) === "[object Array]";
+  return Object.prototype.toString.call(value) === "[object Array]";
```

Comment:
this code won't ship in a version of Firefox that does not have `Array.isArray` now, so we can just remove the funciton after `Array.isArray || `